### ISSUE TITLE
Usability of left section and respective content

### ISSUE THREAD

**Issue by agnimitrapathak** [uber-web/baseui#1743]
Here is the scenario. You can run through the following steps:

1. Land on the site https://baseweb.design
2. Click on **Usage** tab in the left section
3. You notice the content section changes to the "**Base Web Quickstart**"
4. Scroll the page to say **Date Picker**
5. The content section shows date picker but here is the issue _The left section is no longer in view of the user. So to go to say, **File Uploader** section user needs to scroll down along with the content section_

In short the nav section on left needs an independent scroll to keep user on that view. Take a look at https://www.framer.com/api/
